README.md

```diff
@@ -4,12 +4,71 @@
 
 # Identity Wallet
 
-```
-npm run storybook
-```
+This is a wallet built to hold OpenAttestation-compatible documents
+
+https://openattestation.com/docs/getting-started
+https://github.com/Open-Attestation/open-attestation
 
 ## Deployed Application
+
 ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/@dlt/identitywallet)
 
 ## Deployed Storybook
+
 ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/@dlt/identitywallet?release-channel=storybook-default)
+
+# Development
+
+### Install required dependencies
+
+```
+npm i
+```
+
+### To run storybook
+
+```
+npm run storybook
+```
+
+### To run application
+
+```
+npm start
```

Comment:
Would this deserve a bit more ? because you need to scan the qr code with expo app, so either explain, either link to a doc

Also there is a pre-requiite to have expo app